src/_11ty/collections/blogposts.js

```diff
@@ -5,8 +5,15 @@ const {
 module.exports = (collection) => {
 
     let now = new Date();
+    const CONTEXT = process.env.CONTEXT;
 
-  return collection.getFilteredByGlob("./src/content/blog/*.md").filter((item) => {
-            return !item.data.draft && item.date <= now;
-        }).reverse();
+    // for local development and deploy previews, show drafts
+    const drafts = !CONTEXT || CONTEXT === "deploy-preview"
+        ? collection.getFilteredByGlob("./src/content/drafts/*.md")
+            .filter(item => !item.inputPath.includes("README.md"))
+        : [];
```

Comment:
This will exclude drafts from the blog page, but they will still be published and included in the sitemap (and therefore probably indexed).

Perhaps they should also be excluded from publishing, in `.eleventy.js`:

```js
if (process.env.CONTEXT && process.env.CONTEXT !== "deploy-preview") {
    eleventyConfig.ignores.add("src/content/drafts/");
}
```

Reply:
Sounds good. 👍